frontend/app/src/components/defi/LoanInfo.vue

```diff
@@ -0,0 +1,387 @@
+<template>
+  <v-row v-if="!!vault" class="loan-info">
+    <v-col cols="12">
+      <v-row>
+        <v-col class="loan-info__header">
+          <div class="loan-info__header__identifier">
+            Maker Vault #{{ vault.identifier }} ({{ vault.collateralType }})
+          </div>
+          <div class="loan-info__header__owner secondary--text text--lighten-2">
+            Owned by:
+            <hash-link
+              :text="vault.owner"
+              class="d-inline font-weight-medium"
+            ></hash-link>
+          </div>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12" md="6">
+          <stat-card title="Collateral">
+            <div class="d-flex justify-space-between font-weight-medium">
+              <div>
+                Locked up collateral
+              </div>
+              <div>
+                <amount-display
+                  :value="vault.collateralAmount"
+                  :asset="vault.collateralAsset"
+                ></amount-display>
+              </div>
+            </div>
+            <div class="text-right">
+              <div>
+                <amount-display
+                  :value="vault.collateralUsdValue"
+                  fiat-currency="USD"
+                  show-currency="ticker"
+                ></amount-display>
+              </div>
+            </div>
+            <v-divider class="my-4"></v-divider>
+            <div class="d-flex justify-space-between mb-2 font-weight-medium">
+              <div>
+                Current ratio
+              </div>
+              <div>
+                {{
+                  vault.collateralizationRatio
+                    ? vault.collateralizationRatio
+                    : 'NaN'
```

Comment:
Why NaN? I don't think a user should ever see a NaN on the UI. maybe a `-` or sometrhing else?

Reply:
In the implementation you and Lef did liquidation price comes up like this:
![image](https://user-images.githubusercontent.com/27592/84068282-0673e280-a9c9-11ea-81a8-d49c05e3796e.png)

So I assumed you wanted it to be consistent. If we want to override "NaN" with something more "legible" we can do it via `amount-display` (I'm not averse to it).

Reply:
That reminds me:

- [ ] User amountdisplay to display collateralization ratio.

Reply:
if it comes as a `NaN` this is a bug :) it should not come as a `NaN`  basically it was not coming up as a NaN when I did the initial implementation.

Reply:
To me NaN feels like a problem with JavaScript